conversion.go

```diff
@@ -53,7 +53,7 @@ func BatchNewLeafNode(nodesValues []BatchNewLeafNodeData) []LeafNode {
 			var poly [NodeWidth]Fr
 			poly[0].SetUint64(1)
 			for i, nv := range nodesValues {
-				StemFromBytes(&poly[1], nv.Stem)
+				FromLEBytes(&poly[1], nv.Stem)
```

Comment:
I discovered that `StemFromBytes` did the same job as the existing `FromLEBytes`. I think that was a bit confusing since the name seems to imply it does something _special_ related to stems, but in reality, the method doesn't. It is just transforming LE-bytes into a Fr. 

I'll make a PR comment on the exact place where `StemFromBytes` was deleted so we can inspect it a bit better. I'm just mentioning this since you might see other `StemFromBytes -> FromLEBytes` until reaching that point.